packages/cli/src/resource/strings.json

```diff
@@ -28,6 +28,23 @@
         "force": "Force upgrade the project to work with the latest version of Teams Toolkit."
       },
       "success": "Upgrade project successfully."
+    },
+    "doctor": {
+      "account": {
+        "SideLoadingDisabled": "Your Microsoft 365 tenant admin hasn't enabled sideloading permission for your account. You can't install your app to Teams!",
+        "NotSignIn": "No Microsoft 365 account login",
+        "SignInSuccess": "Microsoft 365 Account (%s) is logged in and sideloading permission enabled."
+      },
+      "node" : {
+        "NotFound": "Cannot find Node.js. Go to https://nodejs.org/about/releases/ to install LTS Node.js.",
+        "NotSupported": "Node.js (%s) is not the officially supported version (%s). Your project may continue to work but we recommend to install the supported version.",
```

Comment:
NotFound: `Cannot find Node.js. Node.js used for developing Teams apps with JavaScript or TypeScript using Teams Toolkit for Visual Studio Code. Visit https://nodejs.org/about/releases/ to install the LTS version.`